src/controllers/services.js

```diff
@@ -0,0 +1,101 @@
+import { fetchCollections } from '../services/collections.js';
+import { fetchServices } from '../services/services.js';
+
+export const getServices = async (req, res) => {
+  const { name, termsTypes } = req.query;
```

Comment:
🧠  Later in the code, I find it hard to keep in mind that those values are user-provided.
```suggestion
  const { requestedName: name, requestedTermsTypes: termsTypes } = req.query;
```